doc/usage.rst

```diff
@@ -138,6 +138,26 @@ The *key-string* is a string of 16 characters which is used to
 encrypt each file of Python byte-code before it is stored in
 the archive inside the executable file.
 
+.. _extracting files:
+
+Extracting Files Added to Bundle
```

Comment:
This sounds like one has to extract the files manually. What about "Defining the Extraction Location"?